media_driver/agnostic/common/os/mos_os_virtualengine.cpp

```diff
@@ -94,6 +94,7 @@ MOS_STATUS Mos_VirtualEngineInterface_Initialize(
     }
 
 finish:
+    MOS_SafeFreeMemory(pVEInterf);
```

Comment:
this free() changes the original logic.
if programe reachs here without error, pVEInterf shouldn't be freeed.
need to find where this allocated buffer is freeed somewhere else.